cowprotocol/cowswap
# Orders cancellation from Limit orders page

# Summary

1. Enabled creation of limit orders with an arbitrary expiry date
2. Cancelling of pending orders from the orders table

<img width="852" alt="image" src="https://user-images.githubusercontent.com/7122625/203759052-fc82048e-7235-4fb1-8564-cd9d363687bc.png">

<img width="542" alt="image" src="https://user-images.githubusercontent.com/7122625/203759106-e55c3ddf-2b3f-4652-be18-40706e108646.png">

  # To Test

1. Create an order
2. Open "Open orders" tab
3. Click to the trash icon
- [ ] Order cancelation confirm modal opens
4. Click "Request cancelation"
- [ ] Request to a wallet for sign cancelation
- [ ] Order status changes to "Cancelling"
